src/clparser_perftest.cc

```diff
@@ -155,3 +155,13 @@ int main(int argc, char* argv[]) {
 
   return 0;
 }
+
+#ifdef _WIN32
+int wmain(int argc, wchar_t** wargv) {
+  return mainUTF8(argc, convertCommandLine(argc, wargv));
+}
+#else
+int main(int argc, char** argv) {
+  return mainUTF8(argc, argv);
```

Comment:
Why have a separate `mainUTF8` ?

Seems to me like `int main()` should stay the same, and `int wmain()` can just call `int main()`.

Reply:
To avoid confusion.

Reply:
What confusion?

Windows is the problematic platform. All the other platforms are already non-wchar_t. It's confusing that non-windows platforms have their main() function call some other main() function.

Reply:
Good point.